src/packages/client/src/__tests__/atomicOperationsUpdate.test.ts

```diff
@@ -33,12 +38,8 @@ describe('minimal atomic update transformation', () => {
       mutation {
         updateOneUser(
           data: {
-            countFloat: {
-              set: 3.1415926
-            }
-            countInt1: {
-              set: 3
-            }
+            countFloat: 3.1415926
+            countInt1: 3
```

Comment:
Set is not required anymore?

Reply:
Correct! Dom does that now already in the query engine. We got rid of all transformations.